pkg/console/console.go

```diff
@@ -83,13 +86,29 @@ func Run() error {
 
 func webService(handlers service.Service) *restful.WebService {
 	ws := new(restful.WebService)
-	ws.Route(ws.GET(urlPathPrefix + "/token").
+
+	ws.Path("/apis/token.kubevirt.io/v1alpha1")
```

Comment:
Can you extract `v1alpha1` into a shared const somewhere?

Reply:
done